lotus/sem_ops/cascade_utils.py

```diff
@@ -93,7 +110,8 @@ def precision(pos_threshold: float, neg_threshold: float, sorted_pairs: list[tup
     )
     corrected_recall_target = min(1, corrected_recall_target)
     tau_neg_prime = max(
-        x[0] for x in sorted_pairs[::-1] if recall(best_combination[0], x[0], sorted_pairs) >= corrected_recall_target
+        (x[0] for x in sorted_pairs[::-1] if recall(best_combination[0], x[0], sorted_pairs) >= corrected_recall_target),
```

Comment:
Could this be moved to a helper function? Seems like the logic is the same as a few lines up